{"metadata":{"symbolKind":"enum","roleHeading":"Enumeration","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"kind":"identifier","text":"NfinityCloudError"}],"modules":[{"name":"PasswordAndNfinityFramework"}],"externalID":"s:27PasswordAndNfinityFramework0C10CloudErrorO","role":"symbol","title":"NfinityCloudError","navigatorTitle":[{"kind":"identifier","text":"NfinityCloudError"}]},"hierarchy":{"paths":[["doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework"]]},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["Mac Catalyst"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"NfinityCloudError","kind":"identifier"}]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"variants":[{"paths":["\/documentation\/passwordandnfinityframework\/nfinityclouderror"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"symbol","sections":[],"abstract":[{"text":"Represents an enum hierarchy for errors related to the Nfinity Cloud.","type":"text"}],"identifier":{"url":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError","interfaceLanguage":"swift"},"topicSections":[{"identifiers":["doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/attemptExceeded","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/cloudError(errorCode:errorMessage:)","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/cloudHTTPError(httpCode:response:)","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/httpDeserialization(httpCode:cause:headers:)","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/network(cause:)"],"title":"Enumeration Cases"},{"identifiers":["doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/Error-Implementations"],"generated":true,"title":"Default Implementations"}],"relationshipsSections":[{"kind":"relationships","identifiers":["doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/s5ErrorP","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/s8SendableP"],"type":"conformsTo","title":"Conforms To"}],"references":{"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/FaceDetectionSender/sendFaceDetection(imageData:)":{"abstract":[{"text":"Method for sending face detection data, and returns an ","type":"text"},{"code":"Swift.Result","type":"codeVoice"},{"text":" instance.","type":"text"}],"title":"sendFaceDetection(imageData:)","role":"symbol","url":"\/documentation\/passwordandnfinityframework\/facedetectionsender\/sendfacedetection(imagedata:)","fragments":[{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"sendFaceDetection"},{"text":"(","kind":"text"},{"text":"imageData","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"Data","kind":"typeIdentifier","preciseIdentifier":"s:10Foundation4DataV"},{"kind":"text","text":") "},{"text":"async","kind":"keyword"},{"text":" -> Swift","kind":"text"},{"text":".","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:s6ResultO","text":"Result"},{"text":"<","kind":"text"},{"preciseIdentifier":"s:27PasswordAndNfinityFramework18FaceDetectionStateO","kind":"typeIdentifier","text":"FaceDetectionState"},{"kind":"text","text":", "},{"kind":"typeIdentifier","text":"NfinityCloudError","preciseIdentifier":"s:27PasswordAndNfinityFramework0C10CloudErrorO"},{"text":">","kind":"text"}],"kind":"symbol","type":"topic","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/FaceDetectionSender\/sendFaceDetection(imageData:)"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError":{"type":"topic","navigatorTitle":[{"text":"NfinityCloudError","kind":"identifier"}],"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError","abstract":[{"type":"text","text":"Represents an enum hierarchy for errors related to the Nfinity Cloud."}],"title":"NfinityCloudError","url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror","role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"NfinityCloudError"}],"kind":"symbol"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/httpDeserialization(httpCode:cause:headers:)":{"url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/httpdeserialization(httpcode:cause:headers:)","kind":"symbol","abstract":[{"type":"text","text":"Indicates a problem with deserializing the HTTP response into a model object."}],"fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"httpDeserialization","kind":"identifier"},{"text":"(","kind":"text"},{"text":"httpCode","kind":"externalParam"},{"kind":"text","text":": "},{"preciseIdentifier":"s:Si","text":"Int","kind":"typeIdentifier"},{"kind":"text","text":", "},{"text":"cause","kind":"externalParam"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:s5ErrorP","kind":"typeIdentifier","text":"Error"},{"kind":"text","text":", "},{"kind":"externalParam","text":"headers"},{"text":": [","kind":"text"},{"text":"AnyHashable","kind":"typeIdentifier","preciseIdentifier":"s:s11AnyHashableV"},{"text":" : ","kind":"text"},{"kind":"keyword","text":"Any"},{"text":"])","kind":"text"}],"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/httpDeserialization(httpCode:cause:headers:)","type":"topic","role":"symbol","title":"NfinityCloudError.httpDeserialization(httpCode:cause:headers:)"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/cloudError(errorCode:errorMessage:)":{"title":"NfinityCloudError.cloudError(errorCode:errorMessage:)","role":"symbol","url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/clouderror(errorcode:errormessage:)","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/cloudError(errorCode:errorMessage:)","type":"topic","kind":"symbol","abstract":[{"type":"text","text":"Represents an error with a specific error code and message that the Nfinity Cloud might return, but not from network call."}],"fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"cloudError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"errorCode","kind":"externalParam"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":"?, "},{"text":"errorMessage","kind":"externalParam"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":"?)"}]},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/s5ErrorP":{"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/s5ErrorP","type":"unresolvable","title":"Swift.Error"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/attemptExceeded":{"abstract":[{"type":"text","text":"Case represents too many tries on "},{"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/FaceDetectionSender\/sendFaceDetection(imageData:)","isActive":true,"type":"reference"},{"type":"text","text":"."}],"title":"NfinityCloudError.attemptExceeded","role":"symbol","url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/attemptexceeded","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"attemptExceeded","kind":"identifier"}],"kind":"symbol","type":"topic","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/attemptExceeded"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/s8SendableP":{"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/s8SendableP","type":"unresolvable","title":"Swift.Sendable"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/network(cause:)":{"kind":"symbol","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"network","kind":"identifier"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"cause"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:21CommunicatorFramework0A5ErrorO","text":"CommunicatorError"},{"text":")","kind":"text"}],"type":"topic","url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/network(cause:)","abstract":[{"text":"Case represents network error","type":"text"}],"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/network(cause:)","role":"symbol","title":"NfinityCloudError.network(cause:)"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/cloudHTTPError(httpCode:response:)":{"kind":"symbol","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"cloudHTTPError","kind":"identifier"},{"kind":"text","text":"("},{"text":"httpCode","kind":"externalParam"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:Si","text":"Int"},{"kind":"text","text":", "},{"text":"response","kind":"externalParam"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"NfinityCloudErrorResponse","preciseIdentifier":"s:27PasswordAndNfinityFramework0C18CloudErrorResponseV"},{"kind":"text","text":")"}],"type":"topic","url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/cloudhttperror(httpcode:response:)","abstract":[{"text":"Represents an HTTP error from the Nfinity Cloud with the associated HTTP status code and a custom response object.","type":"text"}],"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/cloudHTTPError(httpCode:response:)","role":"symbol","title":"NfinityCloudError.cloudHTTPError(httpCode:response:)"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework":{"abstract":[],"title":"PasswordAndNfinityFramework","url":"\/documentation\/passwordandnfinityframework","type":"topic","role":"collection","kind":"symbol","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/Error-Implementations":{"kind":"article","type":"topic","url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/error-implementations","abstract":[],"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/Error-Implementations","role":"collectionGroup","title":"Error Implementations"}}}